api/src/services/notifications/groups/notify-user-of-membership.ts

```diff
@@ -0,0 +1,30 @@
+import { Group, User, Notification } from "@/models"
+
+import BaseService from "@/services/base-service"
+import { Notifications } from "@/services"
+
+export class NotifyUserOfMembership extends BaseService {
+  constructor(
+    private user: User,
+    private group: Group,
+    private currentUser: User
+  ) {
+    super()
+  }
+
+  async perform() {
+    const safeTitle = Notification.sanitizeAttribute("title", `Group Added: ${this.group.name}`)
```

Comment:
Consider using a "bare word" here.
```suggestion
   const { name } = this.group
    const safeTitle = Notification.sanitizeAttribute("title", `Group Added: ${name}`)
```
or
```
    const { name: groupName } = this.group
    const safeTitle = Notification.sanitizeAttribute("title", `Group Added: ${groupName}`)
```